src/pkg/exporter/project_collector.go

```diff
@@ -28,7 +28,7 @@ import (
 const ProjectCollectorName = "ProjectCollector"
 
 var (
-	totalProjectSQL = `SELECT project_metadata.value AS public, COUNT(project_metadata.value)
+	totalProjectSQL = `SELECT project_metadata.value AS public, COUNT(project_metadata.value) as count
```

Comment:
Please use uppercase `AS` for consistency.

Reply:
@chlins updated.